src/context/favorites/Favorites.test.tsx

```diff
@@ -40,6 +40,24 @@ describe('Favorites context', () => {
     expect(localStorage.getItem('favorites')).toBe(JSON.stringify([city2, city1]));
   });
 
+  it('should remove single city', () => {
+    const { result } = renderHook(() => useFavorites(), { wrapper: ProvidersCombined });
+
+    act(() => {
+      result.current && result.current.dispatch(add(city1));
+      result.current && result.current.dispatch(add(city2));
+    });
+
+    expect(result.current?.state.favorites).toMatchObject([city1, city2]);
+
+    act(() => {
+      result.current && result.current.dispatch(remove(city1));
+    });
+
+    expect(result.current?.state.favorites).toMatchObject([city2]);
+    expect(localStorage.getItem('favorites')).toBe(JSON.stringify([city2]));
```

Comment:
please clear `localStorage` in `afterEach`

Reply:
We clear `localStorage` with `afterEach` in `setupTests.ts` file